cfapi/__init__.py

```diff
@@ -26,7 +26,6 @@ def get_brigades(official_brigades_only=False):
             else:
                 brigades.append(org)
 
-    brigades = json.dumps(brigades)
```

Comment:
Heh, that was weird that we returned a string from that method. Good improvement 👍 .